waybackpy/cli.py

```diff
@@ -218,41 +218,41 @@ def save_urls_on_file(url_gen: Generator[str, None, None]) -> None:
     + "will be printed.",
 )
 def main(  # pylint: disable=no-value-for-parameter
-    url: Optional[str],
     user_agent: str,
     version: bool,
     show_license: bool,
     newest: bool,
     oldest: bool,
     json: bool,
     near: bool,
-    year: Optional[int],
-    month: Optional[int],
-    day: Optional[int],
-    hour: Optional[int],
-    minute: Optional[int],
     save: bool,
     headers: bool,
     known_urls: bool,
     subdomain: bool,
     file: bool,
     cdx: bool,
-    start_timestamp: Optional[str],
-    end_timestamp: Optional[str],
     cdx_filter: List[str],
-    match_type: Optional[str],
-    gzip: Optional[str],
     collapse: List[str],
-    limit: Optional[str],
     cdx_print: List[str],
+    url: Optional[str] = None,
+    year: Optional[int] = None,
+    month: Optional[int] = None,
+    day: Optional[int] = None,
+    hour: Optional[int] = None,
+    minute: Optional[int] = None,
+    start_timestamp: Optional[str] = None,
+    end_timestamp: Optional[str] = None,
+    match_type: Optional[str] = None,
+    gzip: Optional[str] = None,
+    limit: Optional[str] = None,
 ) -> None:
     """\b
                          _                _
                         | |              | |
     __      ____ _ _   _| |__   __ _  ___| | ___ __  _   _
-    \ \ /\ / / _` | | | | '_ \ / _` |/ __| |/ / '_ \| | | |
-     \ V  V / (_| | |_| | |_) | (_| | (__|   <| |_) | |_| |
-      \_/\_/ \__,_|\__, |_.__/ \__,_|\___|_|\_\ .__/ \__, |
+    \\ \\ /\\ / / _` | | | | '_ \\ / _` |/ __| |/ / '_ \\| | | |
+     \\ V  V / (_| | |_| | |_) | (_| | (__|   <| |_) | |_| |
```

Comment:
The ASCII art appears broken.

Correct art is:

```
                         _                _
                        | |              | |
    __      ____ _ _   _| |__   __ _  ___| | ___ __  _   _
    \ \ /\ / / _` | | | | '_ \ / _` |/ __| |/ / '_ \| | | |
     \ V  V / (_| | |_| | |_) | (_| | (__|   <| |_) | |_| |
      \_/\_/ \__,_|\__, |_.__/ \__,_|\___|_|\_\ .__/ \__, |
                    __/ |                     | |     __/ |
                   |___/                      |_|    |___/

```

Reply:
Why https://github.com/akamhy/waybackpy/pull/135/commits/d2a3946425ba550a79ad22bae96954024dd8fd2a?

Reply:
Pylint says `Invalid escape sequence '\ ' (W605)`...

Reply:
banner in `waybackpy --help` is no problem.

Reply:
> banner in `waybackpy --help` is no problem.

Okay if `waybackpy --help` outputs it fine then merge it.